src/jit/optimizer.cpp

```diff
@@ -3259,10 +3239,6 @@ void Compiler::optUnrollLoops()
                 phdr->gtPrev     = init;
                 head->bbJumpKind = BBJ_NONE;
                 head->bbFlags &= ~BBF_NEEDS_GCPOLL;
-
-                /* Update bbRefs and bbPreds */
-
-                fgRemoveRefPred(head->bbJumpDest, head);
```

Comment:
Why is this no longer necessary? Was it causing trouble when the jump dest for the block was also its immediate successor?